<?xml version="1.0"?>
<launch>
    <node name="object_detector" pkg="object_detector_ros" type="object_detector.py">
        <rosparam command="load" file="$(find object_detector_ros)/launch/params.yaml" />
    </node>
</launch>
